model/legacy-private/src/main/java/org/keycloak/storage/GroupStorageManager.java

```diff
@@ -71,6 +72,28 @@ public Stream<GroupModel> searchForGroupByNameStream(RealmModel realm, String se
         return Stream.concat(local, ext);
     }
 
+    /**
+     * Returns the group hierarchy with the given attribute name and attribute value for the given realm.
```

Comment:
Should we have this Javadoc here? It seems unnecessary since we have it in the interface.

Reply:
Removed javadoc